### ISSUE TITLE
ImageNet pre-trained dynamic model

### ISSUE THREAD

**Issue by devinxzhang** [liyunsheng13/DRT#8]
Hi yunsheng,

Thanks for your interesting work! I am wondering that, for the experiment on DomainNet, did you first pretrain the model on the ImageNet, then finutune on DomainNet? Thanks!

**Comment by liyunsheng13** [comment]
Yes.

**Comment by devinxzhang** [comment]
> 

Is pre-trained dynamic ResNet50 model available?

**Comment by liyunsheng13** [comment]
Unfortunately, we do not have ResNet50 pretrained with DRT.

**Comment by xiaoachen98** [comment]
> Unfortunately, we do not have ResNet50 pretrained with DRT.
Thanks for your interesting work!
Could you provide the top-1 accuracy of ResNetDy you proposed on ImageNet1K dataset？
